src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -352,6 +366,28 @@ public Response updateItem(UUID itemId, Item payload) throws Exception {
     if (!isRealmUser()) {
       return createForbidden("Anonymous users can not update items");
     }
+    
+Currency priceCurrency = getPriceCurrency(payload.getUnitPrice());
+    
+    if (payload.getTypeOfBusiness() == TypeOfBusinessEnum.SALE) {
+      if (payload.getPaymentMethods() == null) {
+        return createBadRequest("PaymentMethods is required");
+      }
+      
+      if (payload.getUnitPrice() == null) {
+        return createBadRequest(String.format("Price is required"));
+      }
+      
+      if (payload.getUnitPrice().getPrice() == null) {
+        return createBadRequest(String.format("Price is required"));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: String contains no format specifiers. ([squid:S3457](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS3457))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=47&issues=AWZVH_tLBfefbO9HrBx-&open=AWZVH_tLBfefbO9HrBx-)

[ISSUE_KEY]: AWZVH_tLBfefbO9HrBx-